packages/autotools-scripts/src/cli.ts

```diff
@@ -21,13 +26,13 @@ program
 program
 .command('a11y')
 .description('test')
-.option('-p, --path <p>', 'project path')
+.option('-f, --files [pattern]', 'Grep file')
 .option('-i, --impact <i>', 'Only display issues with impact level higher than <i>. Value between 1 (minor) and 4 (critical)')
 .action((options) => {
+  const entry = glob.sync(path.join(packagePath, options.files ? options.files : defaultMetaGlob));
```

Comment:
This is going to look for meta files in the `a11y` directly, which is certainly wrong.